src/main/java/com/example/demo/dao/MatchDAO.java

```diff
@@ -0,0 +1,63 @@
+package com.example.demo.dao;
+
+import com.example.demo.configuration.DBConfig;
+import com.example.demo.model.Match;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import java.sql.*;
```

Comment:
Avoid import all from package.
You force developers to use API that may be not needed.